force-app/main/default/classes/ADDR_Addresses_UTIL_TEST.cls

```diff
@@ -0,0 +1,1341 @@
+        /*Address__c address = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1)[0];
+        Account account = UTIL_UnitTestData_TEST.getMultipleTestAccounts(1, UTIL_Describe_API.getBizAccRecTypeID())[0];
+        TDTM_Runnable.DmlWrapper dmlWrapper = new TDTM_Runnable.DmlWrapper();
+
+        Test.startTest();
+        ADDR_Addresses_UTIL.copyAddressAddrSObj(address, account, null, 'Primary_Address_Type__c', dmlWrapper);
+        Test.stopTest();*/
+
+        //Expecting invalid field error
+    }
+
+    @isTest
+    static void testCopyAddressAddrSObjValidField() {
+        Address__c address = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1)[0];
+        Contact contact = new Contact();
+        TDTM_Runnable.DmlWrapper dmlWrapper = new TDTM_Runnable.DmlWrapper();
+
+        Test.startTest();
+        ADDR_Addresses_UTIL.copyAddressAddrSObj(address, contact, 'Mailing', 'Primary_Address_Type__c', dmlWrapper);
+        Test.stopTest();
+
+        System.assertEquals(address.MailingStreet__c, contact.MailingStreet);
+        System.assertEquals(address.MailingCity__c, contact.MailingCity);
+        System.assertEquals(address.MailingCounty__c, contact.Mailing_County__c);
+        System.assertEquals(address.MailingPostalCode__c, contact.MailingPostalCode);
+        System.assertEquals(address.MailingState__c, contact.MailingState);
+        System.assertEquals(address.MailingCountry__c, contact.MailingCountry);
+        System.assertEquals(address.Geolocation__Latitude__s, contact.MailingLatitude);
+        System.assertEquals(address.Geolocation__Longitude__s, contact.MailingLongitude);
+        System.assertEquals(address.Address_Type__c, contact.get('Primary_Address_Type__c'));
+        System.assertEquals(true, dmlWrapper.objectsToUpdate.size() > 0);
+        System.assertEquals(address.MailingStreet__c, dmlWrapper.objectsToUpdate[0].get('MailingStreet'));
+    }
+
+    @isTest
+    static void testCopyAddressAddrSObjInvalidSourceRecordField() {
+        //To Refactor: we don't have guard clause for this.
+        /*Address__c address = UTIL_UnitTestData_TEST.getMultipleTestAddresses(1)[0];
+        Contact contact = new Contact();
+        TDTM_Runnable.DmlWrapper dmlWrapper = new TDTM_Runnable.DmlWrapper();
+
+        Test.startTest();
+        ADDR_Addresses_UTIL.copyAddressAddrSObj(address, contact, 'Billing', 'Primary_Address_Type__c', dmlWrapper);
+        Test.stopTest();*/
+
+        //Expecting invalid field error
+    }
+
+    /**************************************************************************************************************************
+     ****************************************************** initValidStateCountryCodes ****************************************
+     **************************************************************************************************************************/
+
+    //This method is not quite testable as it is using schema describe.
```

Comment:
Make a note in the actual class. Agreed on testability. This may be a good scenario for an unmanaged unit test.